pkg/execution/state/redis_state/lua/queue/dequeue.lua

```diff
@@ -89,6 +92,11 @@ local function handleDequeueConcurrency(keyConcurrency, keyPartitionSet, partiti
 		end
 	end
 
+	if partitionType ~= 0 then
+		-- If this is not a default partition, we don't need to update the global or account pointer.
```

Comment:
Thanks for adding explanatory comments, @BrunoScheufler -- these will 💯% be helpful when we come back here in the future.